Assignment7.txt

```diff
@@ -0,0 +1,19 @@
+Clarke Littlejohn
+
+1) if i recall correctly ++*P moves the pointer then gets the values, *p++ get  the value in the point the increments it
+   and *++p is not an apoerator that works.
+
+2) I am confused by what this si asking it will go by what it is prendence so to will go from right to left or left to right based on the highest order
+
+3) you are working directly with the value so it is always modifing the number.
+
+4.1)string/char array/
+.2)invalid
```

Comment:
This is actually valid! "xyz" is just an array of characters, so the "xyz"[1] is just accessing "y". Then you just subtract 'y' from it to get 0. This is because, if you can recall, a char is just a value mapped to a character!